src/terra-dev-site/doc/application/how-to.2/change-application-theme.doc.mdx

```diff
@@ -2,20 +2,14 @@
 
 The theme used by the application is set using the `themeName` prop on ApplicationBase.
```

Comment:
```suggestion
The theme used by the application is set using the `themeName` prop on ApplicationBase.

The `themeName` prop represents the css class name of the theme.
```

Reply:
Tweaked the descriptions: https://github.com/cerner/terra-application/pull/25/commits/bbbd86df8667fc13a0b7c0ee3ca9997e9365209f